Data/Mod.hs

```diff
@@ -62,21 +62,21 @@ import GHC.TypeNats (Nat, KnownNat, natVal, natVal')
 -- <https://en.wikipedia.org/wiki/Modular_arithmetic#Integers_modulo_n integers modulo m>,
 -- equipped with useful instances.
 --
--- For example, 3 :: 'Mod' 10 stands for the class of integers
--- congruent to \( 3 \bmod 10 \colon \ldots {}−17, −7, 3, 13, 23 \ldots \)
+-- For example, @3 :: 'Mod' 10@ stands for the class of integers
+-- congruent to \( 3 \bmod 10 \colon \ldots, −17, −7, 3, 13, 23, \ldots \)
```

Comment:
Please revert both changes.

Reply:
What's wrong with the code block? It renders the expression in monospace, making it clear that it's a Haskell expression.

Reply:
Personal aesthetic preferences, I don't like mixing spaces from proportional and monospaced fonts in the same line.